models/DataObject/Data/Link.php

```diff
@@ -452,4 +452,18 @@ public function __toString(): string
     {
         return $this->getHtml();
     }
+
+    /**
+     * https://github.com/pimcore/pimcore/pull/15926
+     * used for non-nullable properties stored with null
+     * @TODO: Remove in Pimcore 12
```

Comment:
Should we provide a command to migrate all data befor e update to Pimcore 12? Maybe create a followup issue. It can be more data types to migrate.

Reply:
We don't need a command for this migration as this will be one time job. You can write and execute a simple script to re-save the data which migrates all the properties, not something we intend to handle it in a generic way.

Reply:
But than we need at least a note in the upgrade notes, that everyone need to create a simple script to save alle objects with link fields. But this is enogh when we remove the method. But for this we can create a followup issue für Pimcore 12.

Reply:
yes, this then required to execute at the latest when upgrading to v12. https://github.com/pimcore/pimcore/issues/15990